lib/azure/armrest/wrapper/base_wrapper.rb

```diff
@@ -0,0 +1,81 @@
+require 'delegate'
+require 'ostruct'
+
+module Azure
+  module ArmRest
+    # Base class for JSON wrapper classes. Each Service class should have
+    # a corresponding class that wraps the JSON it collects, and each of
+    # them should subclass this base class.
+    class BaseWrapper < Delegator
```

Comment:
Can we rename `wrapper` to `model` for both the directory and the class name? `Fog` also uses the term `model` although it is not an  `ActiveRecord`